spec/features/shelters/cannot_delete_with_approved_pets_spec.rb

```diff
@@ -68,4 +71,21 @@
 
       expect(page).to_not have_content(@pet_2.name)
   end
+
+
+  it "deletes the reviews for the shelter when the shelter is deleted" do
+    reviews_count = Review.count
+    expect(reviews_count).to eq(2)
+
+    visit "/shelters"
+      within("#shelter-#{@shelter_2.id}") do
+        click_link "Delete #{@shelter_2.name}"
+      end
+
+    reviews_count = Review.count
+    expect(reviews_count).to eq(1)
+
+    review = Review.first
```

Comment:
Create new variable to call on the only review and make sure that the review does not have the content contained in the deleted review.